python/lsst/daf/butler/core/registry.py

```diff
@@ -115,9 +115,19 @@ def setConfigRoot(cls, root, config, full):
             Repository-specific options that should not be obtained
             from defaults when Butler instances are constructed
             should be copied from `full` to `Config`.
```

Comment:
It's not your change but can be fixed if you wish - ` ``full`` and ``config`` `?